### DISCUSSION TITLE
同步冲突问题

### DISCUSSION THREAD

**Discussion by ghost** [volatile-static/Chartero#128] [Q&A]
![屏幕截图 2024-09-26 094730](https://github.com/user-attachments/assets/51518d52-5f20-402c-b625-8ecbf201926f)
每隔一分钟就要出现，阅读时间的冲突，请问要怎么解决

**Comment by volatile-static** [comment]
你现在**扫描周期**设置了几秒？

**Reply by ghost** [reply]
没有设置，是自动同步

**Reply by volatile-static** [reply]
你可以先改到30s以上，可以缓解这一问题
